Products/ZenPackAdapter/services.py

```diff
@@ -68,6 +67,11 @@ def inner(*args, **kw):
     return inner
 
 
+def get(device=None, component=None):
```

Comment:
probably doesn't need to be here.

Reply:
removed - good catch
I think this was left over from when I was comparing the new zobject devices to the (previous) db object devices.